src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -55,6 +55,12 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <%String name = (String) request.getSession().getAttribute("user");
+      boolean admin = name != null && (name.equals("gaurijain") ||
+      name.equals("beckyqiu") || name.equals("mariorios"));
```

Comment:
Maybe we can create a Set to put/get the admin names? And add functions for adding/deleting admins. It would look a bit better than just hard coding them and comparing one by one.